src/common/image.c

```diff
@@ -1071,12 +1071,8 @@ static int _image_read_duplicates(const uint32_t id, const char *filename)
   }
 
   g_list_free_full(files, g_free);
-
-  return count_xmps_processed;
```

Comment:
This is still suspicious I read on this line:
`-  return count_xmps_processed;`

That is, removing the return for this routine having an int as returned. Is that also a GitHub bug ? I'm confused now....

Reply:
I see, good then I'll remove the return int from spec.

Reply:
Ok, void set back in you commit. I was confused by the clean-up mixed with the new feature, better to have two commits in this case.

Reply:
> better to have two commits in this case.

Sorry. I'll try to do that way next time. It is still not completely clear for me to get clean commits.
Before I had a lot of commits with a lot of cleaning ones. Was a bit a of mess. Now I join everything in the same commit, meaningless corrections disappear but all changes are mixed. Still to find the right term to get one commit per meaningful change...
Thanks